lib/matplotlib/axes/_base.py

```diff
@@ -37,6 +37,11 @@
 is_string_like = cbook.is_string_like
 is_sequence_of_strings = cbook.is_sequence_of_strings
 
+def _sorted_iteritems(a):
```

Comment:
If I remember correctly this file created the black rectangle that surrounds the axes. The rectangle consists of four lines, and without the modifications those four lines were drawn in random order.